src/applications/disability-benefits/all-claims/tests/utils/submit.unit.spec.jsx

```diff
@@ -704,36 +707,329 @@ describe('extractDateParts', () => {
 describe('addForm0781V2', () => {
   const formData = {
     syncModern0781Flow: true,
-    eventTypes: ['eventType1'],
-    events: ['event1'],
-    workBehaviors: ['workBehavior1'],
-    healthBehaviors: ['healthBehavior1'],
-    otherBehaviors: ['otherBehavior1'],
-    behaviorsDetails: 'details',
-    supportingEvidenceReports: ['report1'],
-    supportingEvidenceRecords: ['record1'],
-    supportingEvidenceWitness: ['witness1'],
-    supportingEvidenceOther: 'otherEvidence',
-    supportingEvidenceUnlisted: 'unlistedEvidence',
+    answerCombatBehaviorQuestions: 'true',
+    eventTypes: {
+      combat: true,
+      mst: true,
+    },
+    events: [
+      {
+        location: 'Where did the event happen?',
+        otherReports: 'Other official report type listed here',
+        timing: 'When did the event happen?',
+        reports: {
+          restricted: true,
+          unrestricted: true,
+          police: true,
+          none: true,
+        },
+        agency: 'Name of the agency that issued the report',
+        city: 'report city',
+        state: 'report state',
+        township: 'report township',
+        country: 'USA',
+      },
+      {
+        location: 'Where did the event happen?',
+        otherReports: 'Other official report type listed here',
+        reports: {
+          restricted: true,
+          unrestricted: true,
+          police: false,
+          none: true,
+        },
+        agency: 'Name of the agency that issued the report',
+        city: 'report city',
+        state: 'report state',
+        township: 'report township',
+        country: 'USA',
+      },
+      {
+        location: 'Where did the event happen?',
+        reports: {
+          restricted: true,
+          unrestricted: true,
+          police: true,
+          none: true,
+        },
+        agency: 'Name of the agency that issued the report',
+        city: 'report city',
+        state: 'report state',
+        township: 'report township',
+        country: 'USA',
+      },
+    ],
+    workBehaviors: { reassignement: true },
```

Comment:
Typo, which probably doesn't matter with this sample data.

```suggestion
    workBehaviors: { reassignment: true },
```

Reply:
Good catch. Fixed in [d10ca07](https://github.com/department-of-veterans-affairs/vets-website/pull/35972/commits/d10ca07b25b8f0421a479e4d99f589ec65b26c44).